### DISCUSSION TITLE
Cannot change single settings

### DISCUSSION THREAD

**Discussion by Matis-11** [qdm12/ddns-updater#509] [Q&A] [answered]
I have installed it on synology in docker. Works ok so far.
But when I change e.g. the update periode than I cannot save the settings anymore.
It claims I have to fill all field, which i don't want to as I'm using json for config e.g.
What do I do wrong? How can I save the settings?

![grafik](https://github.com/qdm12/ddns-updater/assets/80908396/4774fa6b-3e3b-404a-9673-3ba3486b9858)

**Comment by qdm12** [comment, marked as answer]
It's just Synology being silly and not allowing you to leave environment variables defined in the Docker image as empty (and thus us the defaults set in the Docker image). What you can do is set every variables according to the defaults described here:

https://github.com/qdm12/ddns-updater/blob/012a6dddcd388c6110e5ef0006e96c2b1ac18577/Dockerfile#L77_L107

You can set TZ to your timezone, for example `Europe/Brussels`.
